test/Frontend/SuppressRemarks.swift

```diff
@@ -1,15 +0,0 @@
-// RUN: %empty-directory(%t)
```

Comment:
I'm unsure what we actually want to do here. There's not really any other remark that would make sense to check here, but then that makes me wonder if `-suppress-remarks` makes sense as an option anyway.